src/cpp/session/modules/clang/DefinitionIndex.cpp

```diff
@@ -580,16 +606,23 @@ void searchDefinitions(const std::string& term,
    // first search translation units we have an in-memory index for
    // (this will reflect unsaved changes in editor buffers)
    TranslationUnits units = rSourceIndex().getIndexedTranslationUnits();
-   for (const TranslationUnits::value_type& unit : units)
+   for (TranslationUnits::iterator it = units.begin(); it != units.end(); ++it)
    {
+      const TranslationUnit& unit = it->second;
+      const std::string& filename = it->first;
+
+      bool hidden = isGeneratedFile(FilePath(filename));
```

Comment:
I'm not sure about this, because of the "unsaved files" business mentioned above.

Reply:
This is probably okay -- users are unlikely to be editing auto-generated files so such files should almost never be unsaved.

Reply:
Good point. Thanks :-)